### ISSUE TITLE
TypeScript string based enums

### ISSUE THREAD

**Issue by ctaggart** [fable-compiler/ts2fable#17]
TypeScript has string based enums. The TypeScript API itself uses a bunch of them. The current ts2fable chokes on them. As I rewrite it using Fable, I'm not sure how to map them yet.

typescript.d.ts contains:
``` ts
    enum ScriptElementKindModifier {
        none = "",
        publicMemberModifier = "public",
        privateMemberModifier = "private",
        protectedMemberModifier = "protected",
        exportedModifier = "export",
        ambientModifier = "declare",
        staticModifier = "static",
        abstractModifier = "abstract",
    }
```

**Comment by alfonsogarciacaro** [comment]
Rigth now ts2fable can interpret type aliases like `type Foo = "Foo" | "Bar"` as [Fable StringEnum](http://fable.io/docs/interacting.html#stringenum-attribute) ([see here](https://github.com/fable-compiler/ts2fable/blob/d5d88c74c4d06b0b9954299d664018b71771d745/ts2fable.js#L791-L793)), maybe we could extend this to TS string based enums.

**Comment by ctaggart** [comment]
This is fixed with #19 now in the develop branch.
